README.md

```diff
@@ -345,24 +345,33 @@ Usage:
 
 ...
 ```
+#### get_relations_by_prefix
+> This replaces the `get_tables_by_prefix` macro. Note that the `get_tables_by_prefix` macro will
+be deprecated in a future release of this package.
 
-#### get_tables_by_prefix ([source](macros/sql/get_tables_by_prefix.sql))
-This macro returns a list of tables that match a given prefix, with an optional
-exclusion pattern. It's particularly handy paired with `union_tables`.
-
-Usage:
+Returns a list of [Relations](https://docs.getdbt.com/docs/api-variable#section-relation)
+that match a given prefix, with an optional exclusion pattern. It's particularly
+handy paired with `union_relations`.
+**Usage:**
 ```
--- Returns a list of tables that match schema.prefix%
-{% set tables = dbt_utils.get_tables_by_prefix('schema', 'prefix') %}
+-- Returns a list of relations that match schema.prefix%
+{% set relations = dbt_utils.get_relations_by_prefix('my_schema', 'my_prefix') %}
 
--- Returns a list of tables as above, excluding any with underscores
-{% set tables = dbt_utils.get_tables_by_prefix('schema', 'prefix', '%_%') %}
+-- Returns a list of relations as above, excluding any with underscores
+{% set relations = dbt_utils.get_relations_by_prefix('my_schema', 'my_prefix', '%_%') %}
```

Comment:
fyi, the `_` matches any single character in the `like` syntax on some (all?) databases - we were bitten by this in dbt-core over here: https://github.com/fishtown-analytics/dbt/issues/1960

So, super minor, but I'd change this example to one that doesn't exhibit this particular bug :p